class-decent-comment.php

```diff
@@ -187,6 +187,11 @@ function query( $query_vars ) {
 		if ( ! empty( $post_id ) ) {
 			$where .= $wpdb->prepare( ' AND comment_post_ID = %d', $post_id );
 		}
+    else {
+      //if no post_id is specified, exclude current post id, since theme should show it elsewhere:
```

Comment:
this would produce unexpected and undesired results for most users of the plugin as it deviates from its standard behavior